packages/plugin-cutout-library-web/src/plugin.ts

```diff
@@ -0,0 +1,139 @@
+import type CreativeEditorSDK from '@cesdk/cesdk-js';
+import {
+  CreativeEngine,
+  UserInterfaceElements,
+  type EditorPlugin
+} from '@cesdk/cesdk-js';
+import {
+  DEFAULT_PLUGIN_CONFIGURATION,
+  PluginConfiguration,
+  getPluginConfiguration
+} from '.';
+import { registerComponents } from './components';
+import {
+  ASSET_SOURCE_ID,
+  ENTRY_ID,
+  I18N,
+  NOTIFICATION_CUTOUT_BLOCK_SELECTED_ID,
+  NOTIFICATION_DIFFERENT_PAGES_ID,
+  NOTIFICATION_SELECT_CUTOUT_BLOCK_ID
+} from './constants';
+import loadAssetSourceFromContentJSON from './loadAssetSourceFromContentJSON';
+
+export function CutoutPlugin({
+  assetBaseUri,
+  addCanvasMenuButton
+}: PluginConfiguration): EditorPlugin {
+  return {
+    initializeUserInterface: ({ cesdk }) => {
+      addCutoutAssetSource(cesdk, { assetBaseUri });
+      if (addCanvasMenuButton) registerComponents(cesdk);
```

Comment:
Could we align this API with what we have done with the BG removal stuff (`locations` etc).

Also, I think I would still register the canvas menu, but let the configuration not add it by default to the "order".